audio-core/src/channel.rs

```diff
@@ -3,8 +3,8 @@
 
 /// The buffer of a single channel.
 ///
-/// This doesn't provide direct access to the underlying buffer, but rather
-/// allows us to copy data usinga  number of utility functions.
```

Comment:
I would rather drop this entire paragraph. Seems outdated and doesn't really tell us anything which isn't obvious.

Reply:
Good point, though leaving this as simply "The buffer of a single channel" is confusing because elsewhere "buffer" means multichannel. So I rewrote that too.